.github/workflows/release.yml

```diff
@@ -59,22 +59,22 @@ jobs:
         id: local_repo
         run: echo ::set-output name=dir::"$(mktemp -d repo.XXXXXXXX)"
       - name: Download Artifacts
-        uses: SonarSource/gh-action_release/download-build@v4
+        uses: SonarSource/gh-action_release/download-build@4.0.1
         with:
           build-number: ${{ steps.get_version.outputs.build }}
           local-repo-dir: ${{ steps.local_repo.outputs.dir }}
       - name: Maven Central Sync
         id: maven-central-sync
         continue-on-error: true
-        uses: SonarSource/gh-action_release/maven-central-sync@v4
+        uses: SonarSource/gh-action_release/maven-central-sync@4.0.1
         with:
           local-repo-dir: ${{ steps.local_repo.outputs.dir }}
         env:
           OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
           OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
       - name: Notify on failure
         if: ${{ failure() || steps.maven-central-sync.outcome == 'failure' }}
-        uses: 8398a7/action-slack@v4
+        uses: 8398a7/action-slack@v3.13.0
```

Comment:
I'm fine with a fixed version here, as we don't have control over that repository.
@tomverin Shall we update the template https://github.com/SonarSource/sonar-dummy-oss/blob/master/.github/workflows/release.yml to avoid security issues?

We might get into trouble if somebody manages to push a commit to that `8398a7/action-slack@v3` branch.